src/components/settings/settings.js

```diff
@@ -134,16 +134,16 @@ class SettingsControl extends BaseComponent {
    * @memberof SettingsControl
    */
   getQualityOptionLabel(t: Object): string {
-    let resolution = t.height ? t.height + 'p' : undefined;
+    let qualityLabel = t.qualityLabel ? t.qualityLabel : t.height ? t.height + 'p' : undefined;
     let mbs = t.bandwidth ? (t.bandwidth / 1000000).toPrecision(2) + 'Mbs' : undefined;
 
     if (!this.props.qualityType) {
-      return resolution || mbs || 'N/A';
+      return qualityLabel || mbs || 'N/A';
     }
     else if (this.props.qualityType.toUpperCase() === 'MBS' && mbs) {
       return mbs;
     }
-    else if (this.props.qualityType.toUpperCase() === 'RESOLUTION' && resolution) {
+    else if (this.props.qualityType.toUpperCase() === 'RESOLUTION' && qualityLabel) {
       return t.height + 'p';
```

Comment:
we already calculate the resolution label above, just like mbs, so need to return  `resolution`.

Reply:
also, resolution was the proper name, as we calculate resolution or bitrate.